src/ui/src/api/api.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+import { IRegisterFormValues } from '../interfaces/Forms/IRegisterFormValues';
+import { ILoginFormValues } from '../interfaces/Forms/ILoginFormValues';
+
+const baseURL = 'https://localhost:5001/api/auth';
```

Comment:
Antipattern. Expose axios client with base URL. + use gateway URL. + grab this value from .env file.
File should be stored in api/rest/api.ts, because we already have `wss` folder